dcp_prototype/backend/browser/rds/load_artifact.py

```diff
@@ -0,0 +1,91 @@
+import os
```

Comment:
This file is a temporary script right? I would probably create a `scripts` directory also noting that this will be likely be deleted/replaced at some point.

Reply:
Moved to `scripts` dir, added comment